### ISSUE TITLE
Accessibility - NVDA Screen Reader for Card + Accordion

### ISSUE THREAD

**Issue by yangster23** [react-bootstrap/react-bootstrap#5652]
## Describe the bug

Using the Accordion Toggle as a card header component, screen readers like NVDA does not announce the state of the controls whether the control is in expanded or collapsed. This violates accessibility guidelines MAS 1.3.1 - Info and Relationship.

## To Reproduce

1) Turn on NVDA 
2) Click/hit ENTER key on entire clickable card header of any card component in the Accordion
3) Observe the announcement by NVDA 

## Reproducible Example

Existing example in https://react-bootstrap.github.io/components/accordion/ - 
![image](https://user-images.githubusercontent.com/5368099/107835507-c0e19380-6d67-11eb-94d3-0609be74f63b.png)

## Expected behavior

When clicking on card header, NVDA should announce the state of the control whether the control is in expanded or collapsed, for example "Clickable Card collapsed" if control is in collapsed state and also the NVDA should announce the state after activating the control, means if we are changing the state from Collapsed to Expand then NVDA should announce the updated state that is "Clickable Card expanded"

## Environment (please complete the following information)

- Operating System: Windows
- Browser, Version:  Chrome 88.0.4324.150, Chromium Edge Version: 88.0.705.18
- Screen reader: NVDA(2020.3), Narrator
- React-Bootstrap Version [e.g. 1.4.0]

## Additional context

Note when using <Accordion.Toggle as={Button} variant="link" eventKey="0" aria-expanded={true}> will satisfy accessibility issue because button has valid aria-expanded property; however, using Accordion Toggle as Button does not meet our UI needs of allowing the entire header to be clickable. Hence, we are needing to use <Accordion.Toggle as={Card.Header}> but inserting aria-expand does not solve issue.

**Comment by kyletsang** [comment]
We can fix the aria-expanded bit here, but seems like NVDA still doesn't announce the collapsed state even with a role set to "button" on the toggle element.  Is this a bug in NVDA?

There are a couple suggestions I have for using a button as the toggle while still having the entire clickable header:

The upstream Bootstrap example has a block button toggle that spans the width of the card header (but the padding isn't clickable):
https://getbootstrap.com/docs/4.6/components/collapse/#accordion-example

Alternatively, you can control the styling by removing the padding from the header and adding it into the button toggle so the whole thing is clickable like so:
https://codesandbox.io/s/elegant-sara-x3fre

**Comment by yangster23** [comment]
Microsoft's screen narrator with Edge gives us the same error here. Regardless, the alternative suggestion you provided should be sufficient! Thanks.

**Comment by marishkaSunrise** [comment]
I have the same issue:  aria-expanded attribute doesn't work with alwaysOpen prop on Accordion.Button. Also aria-controls attribute is missing .
